frontend/tests/e2e/apply/features/happy_path_apply.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Happy Path – Apply Workflow
+  As a grant applicant
+  I want to complete the full Apply workflow successfully
+  So that I can submit a complete application without errors
+
+  Background:
+    Given the user is able to login with all roles needed
+    And the system is available
+
+  @Happy-Path
+  Scenario: Complete the full Apply workflow successfully
+
+    # --- Login ---
+    Given the user launches the URL
+    When the user enters valid login credentials
+    And the user logs in
+    Then the user goes to the search page
+    When the user searches for the specific Funding Opportunity number
+    Then the user is taken to the Funding Opportunities page
+
+    # --- Starting Application ---
+    When the user clicks "Start Application"
+    Then the "Start a new application" modal opens
+    When the user selects a specific option in the "Who's applying" drop down
+    And enters the name of the application
+    And clicks on "Create Application" button
+    Then a new workspace is created
```

Comment:
`workspace` is a very loaded term here - let's say that `a new application is created`